Gemfile

```diff
@@ -59,6 +59,10 @@ gem "rails_semantic_logger"
 gem "sidekiq"
 gem "woothee"
 
+gem "sentry-ruby"
```

Comment:
I think we don't need this line since `sentry-rails` depends on it.

Reply:
49e5ded